### DISCUSSION TITLE
definition中的中文字体无法用\textbf加粗

### DISCUSSION THREAD

**Discussion by ssssylviaaa** [stone-zeng/fduthesis#240] [Q&A] [answered]

**Comment by sikouhjw** [comment]
1. 定理类环境内容用的是楷体
   > 定理内容的字体。中文模板默认为 `\fdu@kai`，即楷体；英文模板默认为 `\itshape`，即斜体。
2. 模板对不同操作系统的楷体处理如下：
   - `adobe`：伪粗
   - `fandol`：全为 `Regular`
   - `founder`：伪粗
   - `mac`：对应粗体
   - `sinotype`：伪粗
   - `windows`：伪粗
3. 请提供你的代码、发行版

**Comment by ssssylviaaa** [comment]
> 1. 定理类环境内容用的是楷体
>    > 定理内容的字体。中文模板默认为 `\fdu@kai`，即楷体；英文模板默认为 `\itshape`，即斜体。
> 2. 模板对不同操作系统的楷体处理如下：
>    
>    * `adobe`：伪粗
>    * `fandol`：全为 `Regular`
>    * `founder`：伪粗
>    * `mac`：对应粗体
>    * `sinotype`：伪粗
>    * `windows`：伪粗
> 3. 请提供你的代码、发行版

![image](https://user-images.githubusercontent.com/37822963/150676714-cea5499d-6dc1-4134-b4b8-eabc55115a96.png)
![image](https://user-images.githubusercontent.com/37822963/150676728-5546689b-6787-4811-8cf0-d52bc4dd9dc7.png)
操作系统是mac

**Comment by sikouhjw** [comment]
> 操作系统是mac

1. 您用的是 Overleaf
2. 只有本地编译才分操作系统
3. 您可以结合 [Fonts for CJK](https://www.overleaf.com/learn/latex/Questions/Which%20OTF%20or%20TTF%20fonts%20are%20supported%20via%20fontspec%3F#Chinese) 自行设置中文字体

**Comment by ssssylviaaa** [comment]
> > 操作系统是mac
> 
> 1. 您用的是 Overleaf
> 2. 只有本地编译才分操作系统
> 3. 您可以结合 [Fonts for CJK](https://www.overleaf.com/learn/latex/Questions/Which%20OTF%20or%20TTF%20fonts%20are%20supported%20via%20fontspec%3F#Chinese) 自行设置中文字体

应该怎么设置呢，就只想对部分中文字体加粗

**Comment by sikouhjw** [comment, marked as answer]
您可以将这段代码放到导言区
```tex
\ExplSyntaxOn
\cs_set_protected:Npn \__fdu_load_cjk_font_fandol:
  {
    \__fdu_setCJKmainfont:nn   { FandolSong }
      {
        Extension      = .otf,
        UprightFont    = *-Regular,
        BoldFont       = *-Bold,
        ItalicFont     = *-Regular,
        BoldItalicFont = *-Bold
      }
    \__fdu_setCJKsansfont:nn   { FandolHei  }
      {
        Extension      = .otf,
        UprightFont    = *-Regular,
        BoldFont       = *-Bold,
        ItalicFont     = *-Regular,
        BoldItalicFont = *-Bold
      }
    \__fdu_setCJKmonofont:nn   { FandolFang }
      {
        Extension      = .otf,
        UprightFont    = *-Regular,
        BoldFont       = *-Regular,
        ItalicFont     = *-Regular,
        BoldItalicFont = *-Regular
      }
    \__fdu_set_cjk_font_kai:n { FandolKai  }
  }
\ExplSyntaxOff
```

**Comment by ssssylviaaa** [comment]
> 您可以将这段代码放到导言区
> 
> ```
> \ExplSyntaxOn
> \cs_set_protected:Npn \__fdu_load_cjk_font_fandol:
>   {
>     \__fdu_setCJKmainfont:nn   { FandolSong }
>       {
>         Extension      = .otf,
>         UprightFont    = *-Regular,
>         BoldFont       = *-Bold,
>         ItalicFont     = *-Regular,
>         BoldItalicFont = *-Bold
>       }
>     \__fdu_setCJKsansfont:nn   { FandolHei  }
>       {
>         Extension      = .otf,
>         UprightFont    = *-Regular,
>         BoldFont       = *-Bold,
>         ItalicFont     = *-Regular,
>         BoldItalicFont = *-Bold
>       }
>     \__fdu_setCJKmonofont:nn   { FandolFang }
>       {
>         Extension      = .otf,
>         UprightFont    = *-Regular,
>         BoldFont       = *-Regular,
>         ItalicFont     = *-Regular,
>         BoldItalicFont = *-Regular
>       }
>     \__fdu_set_cjk_font_kai:n { FandolKai  }
>   }
> \ExplSyntaxOff
> ```

1、导言区具体是哪里呀，麻烦可以截个图吗~
2、加上这段代码之后在定义里还是直接用\textbf命令就可以对部分字体加粗了吗

**Comment by sikouhjw** [comment]
您如果不知道导言区是什么，请阅读 `texdoc lshort-zh-cn`

**Comment by ssssylviaaa** [comment]
> 您如果不知道导言区是什么，请阅读 `texdoc lshort-zh-cn`

可以粗体显示了！非常感谢！！！
